docs/UserGuide.md

```diff
@@ -91,20 +86,65 @@ You can use the sample data pre-loaded in the application to get play around and
 
 </div>
 
-### Viewing help : `help`
+#### **2.2.3 Types of User Input**
+To expand.
 
-* Gives instructions on how to enter orders into the CLI.
-* Displays a message with a list of all available commands and their format.
-* To return to the main order list click on the `Return to the main order list` button. 
-* Help can also be accessed by clicking the `help` button in the top left corner of the application or by clicking the `F1` keyboard key.
+##### `NAME`
+* The name of the person who has made the order.
+* The name of the person must not be empty, it can take in numbers, characters or a mix of both.
```

Comment:
Character by definition includes numbers as well. Maybe you can say it can take in alphanumeric characters?

Reply:
will change to placed